aws-java/pom.xml

```diff
@@ -26,7 +26,7 @@
         <dependency>
             <groupId>com.pulumi</groupId>
             <artifactId>aws</artifactId>
-            <version>(,6.0.0]</version>
+            <version>(,5.9]</version>
```

Comment:
I would use `5.99`. Current AWS provider release is already at 5.42.0.

Reply:
Thanks @ringods for catching this!